automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Clickable.scala

```diff
@@ -22,6 +22,16 @@ trait Clickable extends LazyLogging { this: Component =>
         new WebDriverWait(webDriver, 5).until(ExpectedConditions.elementToBeClickable(query.element.underlying))
     }
     scrollToVisible()
-    click on query
+    try {
+      click on query
+    } catch {
+      case e: WebDriverException =>
+        logger.warn("doClick: " + e.getMessage)
+        if (e.getMessage.contains("Other element would receive the click")) {
+          logger.warn("retrying \"click on query\" after sleep 5 seconds")
+          Thread.sleep(5000)
+          click on query
+        }
+    }
```

Comment:
sometimes click fails b/c spinner were blocking the click. this retry attempts to prevent test from fail due to this scenario.